spec/Reorder/ReorderServiceSpec.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Sylius\CustomerReorderPlugin\Reorder;
+
+use Doctrine\ORM\EntityManagerInterface;
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Order\Processor\OrderProcessorInterface;
+use Sylius\CustomerReorderPlugin\Factory\OrderFactoryInterface;
+use Sylius\CustomerReorderPlugin\Reorder\ReorderService;
+use Sylius\CustomerReorderPlugin\Reorder\ReorderServiceInterface;
+
+final class ReorderServiceSpec extends ObjectBehavior
```

Comment:
`*Service` is a lazy name for service 😄 what would you say for `Reorderer`? sounds weird but creative